README.md

```diff
@@ -89,7 +89,7 @@ As the /me endpoints use the GU_U and SC_GU_U from the Cookie request header you
 
 ### Starting the API
 1. To start the Members' data API service run `./start-api.sh`.  
-The service will be running on 9400 and use the SupporterAttributesFallback-DEV DynamoDB table.
```

Comment:
This was very out of date, this dynamo table hasn't been used for a couple of years